ITTN-048.tex

```diff
@@ -22,17 +22,22 @@
 \setDocRef{ITTN-048}
 \setDocUpstreamLocation{\url{https://github.com/lsst-it/ittn-048}}
 
-\date{\vcsDate}
+\date {\today}
 
 % Optional: name of the document's curator
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-The Linux Unified Key Setup (LUKS) is a disk encryption, which implements a platform-independent standard on-disk format for use in various tools.  
-The Policy-Based Decryption (PBD) is a collection of technologies that enable unlocking encrypted root and secondary volumes of hard drives on physical and virtual machines using different methods like a user password, a Trusted Platform Module (TPM) device, a PKCS#11 device connected to a system, for example, a smart card, or with the help of a special network server.
+The Linux Unified Key Setup (LUKS) is a disk encryption, which implements a platform-independent standard on-disk format for use in various tools.
+
+The Policy-Based Decryption (PBD) is a collection of technologies that enable unlocking encrypted root and secondary volumes of hard drives on physical and virtual machines using different methods like a user password, a Trusted Platform Module (TPM) device, a PKCS11 device connected to a system, for example, a smart card, or with the help of a special network server.
+
 The PBD as a technology allows combining different unlocking methods into a policy creating an ability to unlock the same volume in different ways. The current implementation of the PBD in Red Hat Enterprise Linux consists of the Clevis framework and plugins called pins. Each pin provides a separate unlocking capability. For now, the only two pins available are the ones that allow volumes to be unlocked with TPM or with a network server.
-The Network Bound Disc Encryption (NBDE) is a subcategory of the PBD technologies that allows binding the encrypted volumes to a special network server. The current implementation of the NBDE includes Clevis pin for Tang server and the Tang server itself. 
-Based on this tools, the Servers System Disk will we encrypted and when they boot, will request decryption to a centralized server that withholds the Decryption module, avoiding the password prompt at boot. 
+
+The Network Bound Disc Encryption (NBDE) is a subcategory of the PBD technologies that allows binding the encrypted volumes to a special network server. The current implementation of the NBDE includes Clevis pin for Tang server and the Tang server itself.
+
+
+Based on this tools, the Servers System Disk will we encrypted and when they boot, will request decryption to a centralized server that withholds the Decryption module, avoiding the password prompt at boot.
```

Comment:
Based on "these" tools.......will "be" encrypted....."they will request.....

Reply:
fixed